stable/gcloud-sqlproxy/values.yaml

```diff
@@ -53,10 +53,20 @@ cloudsql:
     region: "region"
     # Port number for the proxy to expose for this instance.
     port: 5432
+    # Use private network connection
+  private: false
```

Comment:
Instead of adding this value, an we instead just use `extraArgs` and add `ip_address_types: PRIVATE` ? it can be a comment below `extraArgs` that users can uncomment if they so wish

Reply:
I checked and yes it can be used with extraArgs